src/components/ModalViewContainer/CancelAlert/index.tsx

```diff
@@ -0,0 +1,125 @@
+import React, { useState } from 'react';
```

Comment:
Also, since the Cancel Alert seems to show up in places other than ModalViewContainer, perhaps it should be moved up to be its own component. It should be in the same place as it is in the Angular SDK.